client/components/mma/identity/models.ts

```diff
@@ -598,3 +598,46 @@ export const COUNTRIES: CountryValues[] = [
 	{ iso: 'ZW', name: 'Zimbabwe' },
 	{ iso: 'AX', name: 'Åland Islands' },
 ];
+
+export const nonServiceableCountries: string[] = [
```

Comment:
should this list go in a different file/folder? I'm not sure it's Identity related

Reply:
Moved into it's own file